### PR TITLE
Add nginx hls ip and port to client and add flag for https in nginx

### PR DESCRIPTION
**Pull request by melvinheks**

### CODE REVIEW

File: src/web/client/src/components/LiveStream/LiveStream.tsx [hunk 1]
```diff
@@ -72,9 +77,15 @@ export default class LiveStream extends Component<Props, State> {
    * @params streamId - The ID of a particular stream
    */
   getURL = (streamId: string) => {
-    this.setState({
-      url: `http://${this.state.ip}:${this.state.port}/hls/${streamId}.m3u8`,
-    });
+    if (ENABLE_HTTPS) {
+      this.setState({
+        url: `https://${this.state.ip}/hls/${streamId}.m3u8`,
```
Comment (yliu0715) [inline src/web/client/src/components/LiveStream/LiveStream.tsx:82]: No port here?

Comment (melvinheks) [reply src/web/client/src/components/LiveStream/LiveStream.tsx:82]: https port is automatically 443

Comment (chc5) [inline src/web/client/src/components/LiveStream/LiveStream.tsx:82]: Did you forget to put a port here?

Comment (yliu0715) [reply src/web/client/src/components/LiveStream/LiveStream.tsx:82]: ok

Comment (melvinheks) [reply src/web/client/src/components/LiveStream/LiveStream.tsx:82]: https port is automatically 443

**Comment by chc5** [comment]
I got this as an error when viewing livestream
Screenshot from 2020-04-26 02-26-52

**Pull request merged**